tests/chart_tests/test_authsidecar.py

```diff
@@ -150,20 +151,101 @@ def test_authSidecar_kibana(self, kube_version):
         assert doc["metadata"]["name"] == "release-name-kibana"
         assert "auth-proxy" == doc["spec"]["template"]["spec"]["containers"][1]["name"]
 
-        assert "Service" == jmespath.search("kind", docs[2])
-        assert "release-name-kibana" == jmespath.search("metadata.name", docs[2])
-        assert "ClusterIP" == jmespath.search("spec.type", docs[2])
+        assert "Service" == docs[2]["kind"]
+        assert "release-name-kibana" == docs[2]["metadata"]["name"]
+        assert "ClusterIP" == docs[2]["spec"]["type"]
         assert {
             "name": "auth-proxy",
             "protocol": "TCP",
             "port": 8084,
             "appProtocol": "tcp",
-        } in jmespath.search("spec.ports", docs[2])
+        } in docs[2]["spec"]["ports"]
 
         assert "NetworkPolicy" == docs[3]["kind"]
-        assert [{"namespaceSelector": {"matchLabels": {"network.openshift.io/policy-group": "ingress"}}}] == jmespath.search(
-            "spec.ingress[0].from", docs[3]
+        assert [
+            {"namespaceSelector": {"matchLabels": {"network.openshift.io/policy-group": "ingress"}}},
+        ] == docs[3]["spec"]["ingress"][0]["from"]
+        assert {"port": 8084, "protocol": "TCP"} in jmespath.search("spec.ingress[*].ports[0]", docs[3])
```

Comment:
Let switch this type of check to
```suggestion
        assert any([x['ports'] == [{'protocol': 'TCP', 'port': 8084}] for x in docs[3]['spec']['ingress']])
```
then we can get rid of jmespath in this test.